tests/scalers/azure/azure_pipelines_adv/azure_pipelines_adv_test.go

```diff
@@ -0,0 +1,556 @@
+//go:build e2e
+// +build e2e
+
+package azure_pipelines_adv_test
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/microsoft/azure-devops-go-api/azuredevops"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/build"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/taskagent"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/kubernetes"
+
+	. "github.com/kedacore/keda/v2/tests/helper"
+)
+
+// Load environment variables from .env file
+var _ = godotenv.Load("../../../.env")
+
+const (
+	testName = "azure-pipelines-test"
```

Comment:
Use other namespace than the already e2e test for avoid conflicts 
```suggestion
	testName = "azure-pipelines-demands-test"
```